src/main/java/kitchenpos/menu/domain/MenuRepository.java

```diff
@@ -0,0 +1,9 @@
+package kitchenpos.menu.domain;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface MenuRepository extends JpaRepository<Menu, Long> {
+    Long countByIdIn(List<Long> menuIds);
```

Comment:
```suggestion
    long countByIdIn(List<Long> menuIds);
```
래퍼 클래스를 사용할 필요는 없어보이네요

Reply:
수정했습니다 😀😀